lib/sqlalchemy/ext/asyncio/scoping.py

```diff
@@ -1544,14 +1544,16 @@ def info(self) -> Any:
         return self._proxied.info
 
     @classmethod
-    async def close_all(self) -> None:
+    async def close_all(cls) -> None:
         r"""Close all :class:`_asyncio.AsyncSession` sessions.
 
         .. container:: class_bases
 
             Proxied for the :class:`_asyncio.AsyncSession` class on
             behalf of the :class:`_asyncio.scoping.async_scoped_session` class.
 
+        .. deprecated:: 2.0 The :meth:`.AsyncSession.close_all` method is deprecated and will be removed in a future release.  Please refer to :func:`.session.close_all_sessions`.
+
         """  # noqa: E501
 
         return await AsyncSession.close_all()
```

Comment:
```suggestion
    def close_all(cls) -> None:
        r"""Close all :class:`_asyncio.AsyncSession` sessions.

        .. container:: class_bases

            Proxied for the :class:`_asyncio.AsyncSession` class on
            behalf of the :class:`_asyncio.scoping.async_scoped_session` class.

        .. deprecated:: 2.0 The :meth:`.AsyncSession.close_all` method is deprecated and will be removed in a future release.  Please refer to :func:`.session.close_all_sessions`.

        """  # noqa: E501

        return AsyncSession.close_all()
```

I notice that `./tools/generate_proxy_methods.py` give this proposed changes, but I don't think this is a good changes.
Should I accept it?

Reply:
if you want to do "close_all()" manually, which we might have to here since it's different from other methods, just remove it from the list of methods that are automatically generated.